docs/examples/zoom.ipynb

```diff
@@ -69,6 +69,8 @@
     "    sans.isis.CalibrationFilename: '192tubeCalibration_11-02-2019_r5_10lines.nxs',\n",
     "    sans.isis.Filename[sans.types.SampleRun]: 'ZOOM00034786.nxs',\n",
     "    sans.isis.Filename[sans.types.EmptyBeamRun]: 'ZOOM00034787.nxs',\n",
+    "    sans.isis.SampleOffset: sc.vector([0.0, 0.0, 0.11], unit='m'),\n",
```

Comment:
Are these mostly required because of the IDF mechanism in Mantid/ISIS?
Will the situation be better at ESS with the nexus geometry, or are we expecting the geometry in the files to not be updated for every experiment and often be out of date?

Reply:
I think ECDC *wants* to keep files up to date, but if that will be actually true in practice...